governance.md

```diff
@@ -0,0 +1,164 @@
+
+#### Op1st Community Principles 1.0
+Our mission requires us to use open source principles in ways beyond licensing the code we write, but at the core we are an open source community and want to adhere to the best possible principles for creating an open, welcoming, and inclusive community.
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
+  - This is crucial to fulfilling our mission around open sourcing operations
+2. **Welcoming and respectful** : We intend to be an open, welcoming, and inclusive community.
+Diversity is our strength.
+We have and follow a Code of Conduct.
+  - We currently follow the OpenInfra Foundation Code of Conduct as a member of the OpenInfra Labs.
+3. **Open Design** : We follow a transparent and open process for planning and designing any content, services, and features of Op1st.
+Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+4. **Open Work** : Two of our core tools for successful community building are transparency and accessibility.
+To support that, work and collaboration must be done in public to the greatest degree possible.
+Refer to the section on Community Groups and SIGs in particular for details on how this is performed.
+5. **Truly Open Community** : The best way to build a sustainable open source community is to ensure the community is a cohesive, inclusive, and equitable place where all the voices are heard and anyone can rise to leadership positions.
+
+### Code of Conduct
+
+We are using the [upstream OpenInfra Foundation Code of Conduct](https://openinfra.dev/legal/code-of-conduct) until we can convene the new Community SIG to lead an open effort to create our own, modern code of conduct.
+
+### Community membership
+**Note:** This document is a work in progress, especially these membership guidelines
+
+We are opening the Governance 1.0 directly borrowing from the [Kubernetes community group definitions](https://github.com/kubernetes/community/blob/master/community-membership.md), then reviewing and refining for the Governance 1.1.
+
+This doc outlines the various responsibilities of contributor roles in Op1st.
+The Op1st project is subdivided into subprojects under SIGs.
+Responsibilities for most roles are scoped to these subprojects.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | Sponsored by 2 reviewers and multiple contributions to the project | Op1st GitHub org member|
```

Comment:
What is a "contribution"? This is mentioned below by Misc, and can be defined here to be more than *just* code contributions.

Also, "Active" needs clarification. Does someone stop being a member when they're no longer active? If so, what does active mean?

Reply:
Thanks @rbowen @mscherer @schwesig for these comments and observations. I've made this commit: 279810a to address these issues

One thing I'm curious about is how much our assumptions come into play. For example, although "contributions tracked in GitHub" doesn't actually say "only code or docs really matters", I do agree others will read and draw the that or similar conclusion you are. Makes sense to be more clear anyway.

But it does show a bias of old practices. For example, @mscherer's suggestion about 'we focus on ... git based" for now, will extend in the future presumes we won't be having every single role and contribution interact with a git workflow.

By the contrary, I can imagine the vast majority (>95%) of contributions can be tracked back to directly or implied/inferred by git workflow activity. Take event planning as an example: all of that planning, staffing schedules, messaging indexes, etc. are going to be appearing in the form of issues, PRs, comments, reviews, meeting notes and recordings, and so forth.